examples/biosnoop.rs

```diff
@@ -0,0 +1,136 @@
+use bcc::core::BPF;
```

Comment:
Same comment about citing example in the BCC repo if applicable. See https://github.com/rust-bpf/rust-bcc/blob/master/examples/softirqs.rs#L9